src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.tsx

```diff
@@ -143,85 +141,89 @@ export const SliderPanel = <T extends FieldValues>({
-      </StyledFlexTopCenter>
-      <StyledInputContainer>
-        <StyledFlexTopCenter sx={{ flexGrow: 1, gap: '1.2rem' }}>
-          <Uploader
-            {...commonUploaderProps}
-            setValue={(val: string) => setValue(`${name as string}.minImage`, val)}
-            getValue={() => watch(`${name}.minImage`) || ''}
-          />
-          <InputController
-            {...commonInputProps}
-            name={`${name}.min`}
-            label={t('minValue')}
-            maxNumberValue={max - 1}
-            onChange={handleChangeMinScore}
-            minNumberValue={DEFAULT_SLIDER_MIN_NUMBER}
-          />
-        </StyledFlexTopCenter>
-        <StyledFlexTopCenter sx={{ flexGrow: 1, gap: '1.2rem' }}>
-          <Uploader
-            {...commonUploaderProps}
-            setValue={(val: string) => setValue(`${name as string}.maxImage`, val)}
-            getValue={() => watch(`${name}.maxImage`) || ''}
-          />
-          <InputController
-            {...commonInputProps}
-            name={`${name}.max`}
-            label={t('maxValue')}
-            onChange={handleChangeMaxScore}
-            maxNumberValue={DEFAULT_SLIDER_MAX_VALUE}
-            minNumberValue={min + 1}
-          />
-        </StyledFlexTopCenter>
-      </StyledInputContainer>
-      {hasScores && (
+      {isExpanded && (
         <>
-          <StyledScoresContainer>
-            <Table
-              columns={getStaticHeadRow()}
-              rows={getStaticBodyRow()}
-              orderBy="0"
-              uiType={UiType.Secondary}
+          {isMultiple && (
+            <StyledInputContainer sx={{ mb: theme.spacing(2.4) }}>
+              <InputController
+                control={control}
+                name={`${sliderName}.label`}
+                label={t('sliderLabel')}
+                maxLength={SLIDER_LABEL_MAX_LENGTH}
+              />
+            </StyledInputContainer>
+          )}
+          <StyledInputContainer>
+            <InputController
+              control={control}
+              name={`${sliderName}.minLabel`}
+              label={t('minLabel')}
+              maxLength={SLIDER_VALUE_LABEL_MAX_LENGTH}
             />
-            <Table
-              columns={getHeadCells(min, max)}
-              rows={getTableRows(scores, name)}
-              orderBy="0"
-              uiType={UiType.Secondary}
+            <InputController
+              control={control}
+              name={`${sliderName}.maxLabel`}
+              label={t('maxLabel')}
+              maxLength={SLIDER_VALUE_LABEL_MAX_LENGTH}
             />
-          </StyledScoresContainer>
-          {error && (
-            <StyledBodyMedium color={variables.palette.semantic.error}>
-              {error.message}
-            </StyledBodyMedium>
+          </StyledInputContainer>
+          <StyledFlexTopCenter sx={{ p: theme.spacing(2.4, 0.8) }}>
+            <StyledSlider min={minValue} max={maxValue} value={minValue} marks={marks} disabled />
```

Comment:
just to confirm, is Slider it always disabled?

Reply:
Yes, like the others input types while configuring